### PR TITLE
add double-slash-in-pkgconfig-path check

### PR DESCRIPTION
**Pull request by proyvind**

### CODE REVIEW

File: rpmlint [hunk 1]
```diff
@@ -26,7 +26,7 @@ except:  # Python < 2.7
 
 # 1 instead of 0 here because we want the script dir to be looked up first,
 # e.g. for in-place from tarball or SCM checkout
-sys.path.insert(1, '/usr/share/rpmlint')
+sys.path.insert(0, '/usr/share/rpmlint')
```
Comment (scop) [inline rpmlint]: I suppose this change wasn't supposed to be part of the commit?

Comment (proyvind) [inline rpmlint]: yeah, d'oh!

**Comment by Conan-Kudo** [comment]
@proyvind Could you please rebase this against master?

**Comment by Conan-Kudo** [comment]
This change has been merged in a different form as cbe4592f3ba27ec4550d5932101f4d92af88c2f9.

**Pull request closed without merging**